.drone.yml

```diff
@@ -1267,9 +1262,9 @@ steps:
   image: golang:1.18-alpine
   commands:
   - cd "/go/src/github.com/gravitational/teleport/build.assets/tooling"
-  - go run ./cmd/gh-trigger-workflow -owner ${DRONE_REPO_OWNER} -repo teleport.e -workflow
-    release-linux-arm64.yml -workflow-ref=${DRONE_BRANCH} -input oss-teleport-ref=${DRONE_COMMIT}
-    -input upload-artifacts=false -input oss-teleport-repo="${DRONE_REPO}"
```

Comment:
Is it intentional that we're stripping `upload-artifacts=false` here?  That seems like a change unrelated to the distroless stuff -- resulting from some refactored data flows.

Reply:
The workflow fas a `false` default for this, but it probably should be explicit.